homeassistant/components/hue/v2/light.py

```diff
@@ -39,7 +39,7 @@
     normalize_hue_transition,
 )
 
-EFFECT_NONE = "None"
+EFFECT_NONE = "none"
```

Comment:
If we're already causing a breaking change by changing the state/translation key, shouldn't we use the HA provided `EFFECT_OFF` instead? See the [dev blog entry from back then](https://developers.home-assistant.io/blog/2024/01/23/light-color-mode-changes/#a-special-effect-effect_off-which-means-no-effect--turn-off-effect-has-been-added) or [the dev docs about lights](https://developers.home-assistant.io/docs/core/entity/light/#:~:text=The%20current%20effect.%20Should%20be%20EFFECT_OFF%20if%20the%20light%20supports%20effects%20and%20no%20effect%20is%20currently%20rendered).

HA definition: https://github.com/home-assistant/core/blob/f677b910a6582704f5f8f481abb49bd04c9a353b/homeassistant/components/light/__init__.py#L229